scripts/ceph-website/install-deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
The only overall comment is that it's a drag to have to revisit this as we change node versions, and it seems like it's going to just keep getting out of date again.  I don't know if there's a better solution for setting what version of node to use...some feature-based test or something...but if there is something that's better than "it's pinned to 20 until the next time something breaks" that would be preferable.  It wouldn't surprise me if that's not feasible, but I thought I'd throw it out there.

Reply:
That's a nice idea. Maybe we could leverage [nvm-wrapper](https://www.jenkins.io/doc/pipeline/steps/nvm-wrapper/) in the jenkins. Ideally the plugin should have supported getting the [version from .nvmrc](https://github.com/ceph/ceph.io/blob/main/.nvmrc) which is already there in the ceph.io which I just saw. But since this plugin doesn't have the native .nvmrc support, probably we'll have to just read the node version from the file and install the node. It could be something we can try. 

But I don't know how to install and configure a plugin in the Jenkins Job

Reply:
now that i said all this, probably we could just for now install the nvm in the machine and configure it and that should be fine. Let me have a look.

Reply:
just used nvm here.